src/components/Comments.jsx

```diff
@@ -55,7 +75,7 @@ const Comments = () => {
                     <h4 className="title">{comment.author}</h4>
                     {comment.body}
                     <br></br>
-                    Votes: {comment.votes}
+                    Votes: {comment.votes} <button onClick={() => deleteHandler(comment.comment_id)}>Delete Comment</button>
```

Comment:
At the moment I can delete any comment, needs some validation so that the logged in user can only delete their own comments.